Error while checking build/O2Physics/code-check for 17e06d6aa7ecfc533bb5897063a56af8576246ab at 2026-08-15 13:02:
```
## sw/BUILD/O2Physics-code-check-latest/log
--
========== List of issues found ==========
++ echo 'Found 1 errors and 0 warnings.'
Found 1 errors and 0 warnings.
++ [[ 1 -gt 0 ]]
++ cat /sw/BUILD/b6e091985db5b8f02ad80957ecefdcc0b9c11113/O2Physics-code-check/errors.txt
PWGLF/Tasks/Resonances/cha01710analysis.cxx:304:7: error: do not use 'else' after 'return' [readability-else-after-return,-warnings-as-errors]
++ [[ 0 -gt 0 ]]
++ [[ 1 -gt 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/17465/17e06d6aa7ecfc533bb5897063a56af8576246ab/build_O2Physics_code-check/pretty.html).

--- a/PWGLF/Tasks/Resonances/cha01710analysis.cxx
+++ b/PWGLF/Tasks/Resonances/cha01710analysis.cxx
@@ -386,7 +386,7 @@ struct Cha01710analysis {
       centrality = collision.centFT0M();
     } else if (eventCuts.cfgCentEst == kFT0C) {
       centrality = collision.centFT0C();
-    } 
+    }
 
     histos.fill(HIST("Event/hCentDist"), centrality);
     histos.fill(HIST("Event/hPVzDist"), collision.posZ());

--- a/PWGLF/Tasks/Resonances/cha01710analysis.cxx
+++ b/PWGLF/Tasks/Resonances/cha01710analysis.cxx
@@ -301,7 +301,8 @@ struct Cha01710analysis {
     if (dm < v0Cuts.cfgKsMassWindow) {
       histos.fill(HIST("V0/hMassSelected"), v0.mK0Short(), v0.pt());
       return V0MassRegion::kSignal;
-    } else if (dm > v0Cuts.cfgKsMassWindow) {
+    }
+    if (dm > v0Cuts.cfgKsMassWindow) {
       histos.fill(HIST("V0/hMassSelected"), v0.mK0Short(), v0.pt());
       return V0MassRegion::kSideband;
     }

